style.go

```diff
@@ -90,3 +94,12 @@ func NewFont(size int, name string) *Font {
 func DefaulFont() *Font {
 	return NewFont(12, "Verdana")
 }
+
+func DefaulFill() *Fill {
+	return NewFill("none", "FFFFFFFF", "00000000")
+
+}
+
+func DefaulBoder() *Border {
```

Comment:
There's a typo here, and in every place it's used. DefaultBoder should be DefaultBorder.